src/taco-cli/cli/kit.ts

```diff
@@ -427,7 +427,7 @@ class Kit extends commands.TacoCommandBase {
             logger.log(resources.getString("CommandKitSelectStatusUpdatingPlugins"));
         }
 
-        var downloadOptions: Cordova.ICordovaDownloadOptions = { searchpath: "", noregistry: false, usegit: false, cli_variables: {}, browserify: "", link: "", save: true, shrinkwrap: false };
+        var downloadOptions: Cordova.ICordovaDownloadOptions = { searchpath: undefined, noregistry: undefined, usegit: undefined, cli_variables: {}, browserify: undefined, link: undefined, save: true, shrinkwrap: undefined };
```

Comment:
Good point, I see some changes in the interface as part of Nisheet's PR. I will wait until he merges.